Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1343,6 +1323,39 @@ namespace AzToolsFramework
                 EndRecordManipulatorCommand();
             });
 
+        // surface
+        translationManipulators->InstallSurfaceManipulatorMouseDownCallback(
+            [this, manipulatorEntityIds]([[maybe_unused]] const SurfaceManipulator::Action& action)
+            {
+                BuildSortedEntityIdVectorFromEntityIdMap(m_entityIdManipulators.m_lookups, manipulatorEntityIds->m_entityIds);
+
+                InitializeTranslationLookup(m_entityIdManipulators);
+
+                m_axisPreview.m_translation = m_entityIdManipulators.m_manipulators->GetLocalTransform().GetTranslation();
+                m_axisPreview.m_orientation = QuaternionFromTransformNoScaling(m_entityIdManipulators.m_manipulators->GetLocalTransform());
+
+                // [ref 1.]
```

Comment:
what does this comment mean?

Reply:
Ah this is referring to a comment that is true in every mouse down callback, the comment appears on line 1253 above

```
// [ref 1.] Note: in  BaseManipulator, for every mouse down event, we automatically start
// recording an undo command in case something changes - here we take advantage of that
// knowledge by asking for the current undo operation to add our manipulator entity command to
```

I'll update the comment to read `// see comment [ref 1.] above` to make it a little clearer

(also just noticed the line you called out is `1337` 😁 )